manifests/moco-nodes.pp

```diff
@@ -573,6 +575,8 @@
     't-yosemite-r7-101.test.releng.mdc2.mozilla.com' {
     $aspects          = [ 'low-security', 'staging' ]
     $slave_trustlevel = 'try'
+#    $macos_version = regsubst($::macosx_productversion_major, '\.', '')
+    $worker_type = "gecko-t-osx-${macos_version}-beta"
```

Comment:
I agree the name staging is clearer, and that `-s` would better represent that, however currently other staging worker types are using this naming convention...

* `aws-provisioner-v1/gecko-1-b-win2012-beta`
* `aws-provisioner-v1/gecko-t-win10-64-beta`
* `aws-provisioner-v1/gecko-t-win10-64-gpu-b`
* `aws-provisioner-v1/gecko-t-win7-32-beta`
* `aws-provisioner-v1/gecko-t-win7-32-gpu-b`
* `releng-hardware/gecko-t-linux-talos-b`
* `releng-hardware/gecko-t-osx-1010-beta`

I think it would be useful (outside of this PR) for us to create a document about worker type naming, an explanation of what the staging worker types are, how they should be used, the workflow for testing worker type changes in staging, etc. Having a staging strategy would be useful for devs and consumers to be able to better understand how they should go about using staging workers for testing out worker type changes etc.

We could also rename all of the above worker types to have `-s` suffix, if preferred. I think having a document is a good start though in creating some order/understanding around the staging worker types and having a shared view of how they should be set up.

Reply:
Thanks for the clarification!  And, yes, documentation would be great.